compiler/ilgen/IlValue.cpp

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2017, 2017
```

Comment:
Hm, it's virtually new (when I first created the file it was 100% new :) ) but some code from another file crept in. I'll change the initial date to 2016. Good catch, Charlie!